indico_toolkit/auto_review/auto_reviewer.py

```diff
@@ -1,25 +1,30 @@
-from collections import defaultdict
-from typing import List
+from typing import Dict, List, Callable
 
-from .review_config import ReviewConfiguration
-from .auto_review_functions import (
-    accept_by_confidence,
-    reject_by_confidence,
-    reject_by_min_character_length,
-    reject_by_max_character_length,
-    accept_by_all_match_and_confidence,
-    remove_by_confidence,
-)
+class AutoReviewFunction:
+    """
+    Class for hosting functions to manipulate predictions before sending to 
+    auto review
+
+    Args:
+        function (Callable): method to be invoked when applying reviews
+        labels (List[str]): list of labels to invoke method on. Defaults to all labels
+        kwargs (Dict[str, str]): dictionary containing additional arguments needed in calling function
+    """
+    def __init__(
+        self,
+        function: Callable,
+        labels: List[str] = [],
+        kwargs: Dict[str, str] = {},
+    ):
+        self.function = function
+        self.labels = labels
+        self.kwargs = kwargs
 
+    def apply(self, predictions: List[dict] = []):
```

Comment:
mutable default arguments (list) can have issues, do we need a default argument here?

Reply:
removed default arg